graph_altimeter/scan/iam/__init__.py

```diff
@@ -0,0 +1,309 @@
+"""This defines the logic that expands the IAM policies of a graph produced by
+Atimeter"""
+
+import json
+import os
+import re
+import uuid
+import logging
+
+from policyuniverse.policy import Policy
+from policyuniverse.arn import ARN
+
+from graph_altimeter import InvalidArnError, InvalidArnPattern
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+
+# TODO: Strip the data from this file that is not needed.
```

Comment:
Striping out the unneeded data would mean to maintain our own version of the file. Is it worth it? Should we create tooling for generating this file instead of depending on the parliament's version?

Reply:
I think the best thing to do is the later, that is creating our tool.
So I'm removing the comment.